### PR TITLE
Added constants and made HolidayRepositoryImpl implementation internal

### PR DESCRIPTION
**Pull request by Noddy20**
- Made `HolidayRepositoryImpl` internal in library
- Added some constants

### CODE REVIEW

**Comment by Noddy20** [comment]
Should have made 2 separate commits  :|

File: src/main/kotlin/HolidayRepository.kt [hunk 1]
```diff
@@ -1,3 +1,8 @@
 interface HolidayRepository {
     fun getHolidayList(): List<Holiday>
+
+    companion object {
+        @JvmStatic
+        fun getInstance(): HolidayRepository = HolidayRepositoryImpl()
```
Comment (fumiya-kume) [inline src/main/kotlin/HolidayRepository.kt:6]: 💯

File: src/main/kotlin/HolidayRepositoryImpl.kt [hunk 2]
```diff
@@ -25,6 +25,9 @@ class HolidayRepositoryImpl : HolidayRepository {
     }
 
     companion object {
-        private const val name = "国民の祝日・休日名称"
+        private const val HOLIDAY_LIST_CSV_FILE = "syukujitsu.csv"
```
Comment (fumiya-kume) [inline src/main/kotlin/HolidayRepositoryImpl.kt:28]: 👍

**Review by fumiya-kume — APPROVED** [review]
[LGTMeow](https://lgtmeow.com)

**Comment by fumiya-kume** [comment]
Thank you for your contribution :)

**Pull request merged**